### PR TITLE
More Ninja Hardsuits & Gear Crates

### PR DESCRIPTION
**Pull request by RustingWithYou**
Adds three new hardsuits to the ninja selection - the Einstein Engines Paragon, the Vaurca Combat Exoskeleton and the Tupkala Infiltration Suit.

Relabels the old generic Skrell rig as a Qukala assault hardsuit, and adds Qukala gear crates to the antag uplink.

Change the armor values on Skrell and Vaurca hardsuits to be more usable in normal gameplay. Skrell armor values are decreased slightly to be more in line with the power of other antag hardsuits, while the Vaurca suit has had its laser armor lowered and ballistic armor raised in line with changes to Vaurca Warriors' brute/burn mods at the request of bug lore people.

Removes the AI container from Skrell and Dominian suits given those factions' anti-AI stances, and adds it to the uplink in case people really want to do a gimmick around it anyway.

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

**Review by DreamySkrell — APPROVED** [review]

File: code/modules/clothing/spacesuits/rig/suits/skrell.dm [hunk 1]
```diff
@@ -51,13 +53,13 @@
...
+	name = "tupkala infiltration suit control module"
+	desc = "An old yet reliable suit control module, designed by the Nralakk Federation for clandestine operations. The best of the best, albeit outdated now."
 	icon = 'icons/clothing/rig/tup.dmi'
 	icon_state = "tup_rig"
 	suit_type = "tup suit"
 	armor = list(
-		melee = ARMOR_MELEE_VERY_HIGH,
+		melee = ARMOR_MELEE_MAJOR,
 		bullet = ARMOR_BALLISTIC_RIFLE,
 		laser = ARMOR_LASER_RIFLE,
```
Comment (NonQueueingMatt) [inline code/modules/clothing/spacesuits/rig/suits/skrell.dm:64]: This is some ridiculous armour for a ninja hardsuit.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/clothing/spacesuits/rig/suits/skrell.dm (hunk 1 above)
Comment (RustingWithYou) [reply code/modules/clothing/spacesuits/rig/suits/skrell.dm:64]: There are other ninja rigs with similar or better armor values, and lore-wise it makes sense that the Tups would have high-quality shit. I am not an expert on armor balance however, so I can tone it down if need be.

Comment (RustingWithYou) [reply code/modules/clothing/spacesuits/rig/suits/skrell.dm]: ninja version has been nerfed to be more in line with normal combat hardsuit armor, good armor is saved for the funny inaccessible one

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**